source/text/typesetter.ts

```diff
@@ -47,11 +47,11 @@ export class Typesetter {
         /* Please be aware that all vertices getter return typed views on a big float32array.
         As a consequence do strictly rely on in-place operations only. */
 
-        const padding = fontFace.glyphTexturePadding;
+        // const padding = fontFace.glyphTexturePadding;
         const origin: vec3 = vertices.origin(index);
         vec3.set(origin, pen[0], pen[1], 0.0);
-        origin[0] += glyph.bearing[0] + padding[3];
-        origin[1] += glyph.bearing[1] - glyph.extent[1] + padding[2];
+        origin[0] += glyph.bearing[0]; // + padding[3];
+        origin[1] += glyph.bearing[1] - glyph.extent[1]; // + padding[2];
```

Comment:
remove outcommented code? Also below (line 392).

Reply:
this is some leftovers from another branch... will be removed in line anchor fix.